src/context/WalletProvider/local-wallet.ts

```diff
@@ -0,0 +1,39 @@
+import { KeyManager } from './config'
+/**
+ * Defining storage up here,
+ * so it can be changed to other storage types easily
+ */
+const storage = {
+  set: (key: string, value: string) => localStorage.setItem(key, value),
+  get: (key: string) => localStorage.getItem(key),
+  remove: (key: string) => localStorage.removeItem(key)
+}
+
+const LOCAL_WALLET_TYPE = 'localWalletType'
+const LOCAL_WALLET_DEVICE_ID = 'localWalletDeviceId'
+const LOCAL_NATIVE_WALLET_NAME = 'localNativeWalletName'
```

Comment:
this is unnecessarily restrictive. I'd prefer to storage an object under a single key.

If the future we want to be able to to store an array of paired wallets.